sdk/mobile-center-updates/src/main/java/com/microsoft/azure/mobile/updates/InstallerUtils.java

```diff
@@ -0,0 +1,54 @@
+package com.microsoft.azure.mobile.updates;
+
+import android.content.Context;
+import android.support.annotation.NonNull;
+import android.support.annotation.VisibleForTesting;
+
+import com.microsoft.azure.mobile.utils.MobileCenterLog;
+
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Installer utils.
+ */
+class InstallerUtils {
```

Comment:
Does it work for Amazon App Store? It usually downloads APKs from store and install so it can be considered as local install.